backend/organization/utility/organization.py

```diff
@@ -78,3 +78,10 @@ def is_valid_organization_size(org_size) -> bool:
         if size_option[0] == org_size:
             return True
     return False
+
```

Comment:
Non-blocker and Linting: Two spaces between methods.

Reply:
🙌 Does GitHub suggestion's feature allow you to insert newlines for things like this?

Reply:
I will take a look at mypy for this automate.